driver-core/src/main/com/mongodb/operation/ChangeStreamBatchCursor.java

```diff
@@ -143,11 +161,17 @@ public void remove() {
             }
             wrapped.close();
 
-            if (resumeToken != null) {
-                changeStreamOperation.startAfter(null);
+            changeStreamOperation.startAfter(null);
```

Comment:
startAfter has already been made null.

Reply:
Where has `startAfter` been set to null?

Reply:
Apologies, I thought it was always set to null in the constructor but that may not always be the case.